turbinia/evidence.py

```diff
@@ -259,6 +260,18 @@ class Directory(Evidence):
   pass
 
 
+class CompressedDirectory(Evidence):
+  """A compressed directory containing data."""
+
+  def __init__(self, *args, **kwargs):
+    super(CompressedDirectory, self).__init__(*args, **kwargs)
+    self.copyable = True
+
+  def _postprocess(self):
+    # Compress a given directory and return the compressed directory path.
+    self.local_path = archive.CompressFolder(self.local_path)
```

Comment:
in order for the evidence to be actually saved & passed around, it will need to have source_path set to the actual data.
local_path & mount_path would then point to the postprocessed (I would imagine this is the 'decompressed' version of the evidence?).

Though since #388 is not in yet, this needs to be local_path for now =)

Reply:
I changed this to initialize two new variables in the CompressedDirectory evidence object `self.compressed_directory` and `self.uncompressed_directory`. In the meantime, I am setting the local path to these variables when pre/post process method is called.